Backpack/Calendar/Classes/BPKCalendarCellData.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol BPKCalendarCellData <NSObject>
```

Comment:
Should all cell data provide `foregroundColor` and `backgroundColor`? Isn't that specific to the traffic light version?

I expect this to have a method like `- (void)configureCell:(BPKCalendarDateCell *)cell`

Reply:
Ah yeah - the issue with this is that we don't actually set those things on the cell atm. FSCalendar has a delegate that asks for the background and foreground colours. I wasn't sure how to make that work with the new mechanism 🤔

Reply:
Update: have refactored this. `BPKCalendarCellData` is now empty so has been removed.